tests/std/tests/P0356R5_bind_front/test.cpp

```diff
@@ -92,13 +112,39 @@ constexpr bool test_constexpr() {
     bound1(4);
     assert(value == 234);
 
+    // Test PMFs.
+    Cat cat{"Peppermint"};
+    auto bound2 = bind_front(&Cat::noise, cat); // stores a copy
+    assert(bound2("meow") == "Peppermint says meow");
+    cat.name = "Fluffy";
+    assert(cat.noise("hiss") == "Fluffy says hiss");
+    assert(bound2("purr") == "Peppermint says purr");
+
+    auto bound3 = bind_front(&Cat::noise, &cat); // stores a pointer
+    assert(bound3("MEOW") == "Fluffy says MEOW");
+    cat.name = "Peppermint";
+    assert(bound3("PURR") == "Peppermint says PURR");
+
+    auto bound4 = bind_front(&Cat::noise, ref(cat)); // stores a reference_wrapper, uses LWG-2219
+    assert(bound4("Why do you keep renaming me?") == "Peppermint says Why do you keep renaming me?");
+    cat.name = "Cat";
+    assert(bound4("You can't rename me anymore, Human") == "Cat says You can't rename me anymore, Human");
+
     // Test "perfect forwarding call wrapper" behavior.
     auto bound5 = bind_front(DetectQualifiers{});
     assert(bound5() == "modifiable lvalue");
     assert(as_const(bound5)() == "const lvalue");
     assert(move(bound5)() == "modifiable rvalue");
     assert(move(as_const(bound5))() == "const rvalue");
 
+#if _HAS_CXX23
+    test_movable_only_types();
+#else

```

Comment:
Nitpick: We generally comment the `#else` and `#endif` together (or omit both when they are very small).